Tune flags to build libv8 on armel
 mno-thumb-interwork to allow compilation on default debian armel target (-march=armv4t), see
  http://code.google.com/p/v8/issues/detail?id=590
 FIXED IN GCC 4.4.5 : fno-tree-sink to workaround a gcc armel bug, see http://gcc.gnu.org/bugzilla/show_bug.cgi?id=39604
 STILL APPLY : -Wno-psabi to fix gcc warnings on armel, see http://gcc.gnu.org/bugzilla/show_bug.cgi?id=42748

--- a/SConstruct
+++ b/SConstruct
@@ -202,7 +202,8 @@
       },
       'unalignedaccesses:off' : {
         'CPPDEFINES' : ['CAN_USE_UNALIGNED_ACCESSES=0']
-      }
+      },
+      'CCFLAGS':      ['-Wno-psabi']
     },
     'simulator:arm': {
       'CCFLAGS':      ['-m32'],
--- a/src/arm/macro-assembler-arm.cc
+++ b/src/arm/macro-assembler-arm.cc
@@ -57,9 +57,6 @@
 // We do not support thumb inter-working with an arm architecture not supporting
 // the blx instruction (below v5t).  If you know what CPU you are compiling for
 // you can use -march=armv7 or similar.
-#if defined(USE_THUMB_INTERWORK) && !defined(CAN_USE_THUMB_INSTRUCTIONS)
-# error "For thumb inter-working we require an architecture which supports blx"
-#endif
 
 
 // Using bx does not yield better code, so use it only when required
